src/niscope/metadata/config_addon.py

```diff
@@ -2,4 +2,16 @@
 config_additional_config = {
     'module_version': '1.3.2.dev0',
     'latest_runtime_version_tested_against': '20.0.0',
+    'custom_types': [
+        {
+            'ctypes_type': 'struct_niScope_wfmInfo',
+            'file_name': 'waveform_info',
+            'python_name': 'WaveformInfo'
```

Comment:
It was (and still is) in the config.py file. When adding MeasurementStats to custom_types in config_addon.py, it seems to override the equivalent field in config.py and will not generate WaveformInfo unless these lines are present here as well.

Reply:
I think this is exposing a problem. Information should go either in config.py or in confing_addon.py. We are a bit disorganized on this regard in general.
@texasaggie97 what do you think?

Reply:
In this case, it should be added to internal metadata so that `config.py` will be generated with both entries.

Reply:
It seems like this is still pending.

Reply:
One of the perforce changes I'm currently working on is figuring out where this needs to be added in order to be automatically generated as part of the config.py file. Haven't forgotten about this.

Reply:
To elaborate: In other parts of the nimi-python repository (specifically, in NI-Digital) when new custom types are added, they've just been left in config_addon.py. I'd do the same here with MeasurementStats, but as detailed above, that breaks WaveformInfo unless they're both present in config_addon.py. Since new custom types elsewhere have not been moved into internal perforce metadata, I'm not sure what the procedure to add those and generate them correctly is -- the hapigen documentation is a little lacking on this front, and there's no precedent that I can read over to figure it out. The naive solution would be to just add the MeasurementStats custom type into the python global_config.hapigen, but this doesn't have any effect on the generated code. My assumption is that the base hapigen files shouldn't be modified, but that leaves me completely unsure where to edit the internal metadata to add this new custom type.

Reply:
This is acceptable as is, as per discussions with Mark. Issue #1495 filed to address this problem.